share/qt/Info.plist.in

```diff
@@ -17,10 +17,10 @@
   <string>APPL</string>
 
   <key>CFBundleGetInfoString</key>
-  <string>@CLIENT_VERSION_MAJOR@.@CLIENT_VERSION_MINOR@.@CLIENT_VERSION_REVISION@.@CLIENT_VERSION_BUILD@, Copyright © 2009-@COPYRIGHT_YEAR@ @COPYRIGHT_HOLDERS_FINAL@</string>
+  <string>Copyright © 2009-@COPYRIGHT_YEAR@ @COPYRIGHT_HOLDERS_FINAL@</string>
 
   <key>CFBundleShortVersionString</key>
-  <string>@CLIENT_VERSION_MAJOR@.@CLIENT_VERSION_MINOR@.@CLIENT_VERSION_REVISION@</string>
```

Comment:
Why are you removing @CLIENT_VERSION_REVISION?

Reply:
That is the SHORT version string. This should be used for major releases (in this case we should include the minor release number but the client number is for the regular version string. 


I will update the PR and add better explanations soon.

Reply:
One of the goals in this PR will result in better issue tracking for the macOS builds.

Reply:
> That is the SHORT version string. 

I'd rather use the full version string, to avoid any ambiguity. Shortened version strings have caused issues in the past. See #4029 and #14697.